src/lib/stitching/mergeSchemas.ts

```diff
@@ -48,6 +51,20 @@ export const incrementalMergeSchemas = (testConfig?: any) => {
     }
   }
 
+  if (ENABLE_KAWS_STITCHING) {
```

Comment:
TBH, you can probably skip this env var and always stitch 👍

Reply:
True, I'm just gonna leave it for now as I don't have KAWS setup on production yet

Reply:
Cool - your errors are beast the `.env.testing` doesn't have the URLs set

Reply:
Yeah, I noticed, just set the variable via hokusai but I may have to set a default 🤔

Reply:
Nevermind, I see what you're saying... just updated `.env.test`